Dockerfile.ptso

```
@@ -1,30 +1,34 @@
-# Dockerfile for PTSO Agent A2A Client
-FROM python:3.12-slim
+FROM python:3.13-slim
 
 WORKDIR /app
 
-# Install Node.js for MCP tools
-RUN apt-get update && apt-get install -y curl && \
+# Install Node.js 18 (for npx MCP tools)
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)